src/main/java/it/trade/model/request/TradeItPreviewCryptoOrderRequest.java

```diff
@@ -0,0 +1,83 @@
+package it.trade.model.request;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class TradeItPreviewCryptoOrderRequest extends TradeItRequestWithSession {
+    @SerializedName("accountNumber")
+    @Expose
+    public String accountNumber;
+
+    @SerializedName("orderAction")
+    @Expose
+    public String orderAction;
+
+    @SerializedName("orderQuantity")
+    @Expose
+    public Double orderQuantity;
+
+    @SerializedName("orderPair")
+    @Expose
+    public String orderPair;
+
+    @SerializedName("orderPriceType")
+    @Expose
+    public String orderPriceType;
+
+    @SerializedName("orderExpiration")
+    @Expose
+    public String orderExpiration;
+
+    @SerializedName("orderLimitPrice")
+    @Expose
+    public Double orderLimitPrice;
+
+    @SerializedName("orderStopPrice")
+    @Expose
+    public Double orderStopPrice;
+
+    @SerializedName("orderQuantityType")
+    @Expose
+    public String orderQuantityType;
+
+    public TradeItPreviewCryptoOrderRequest(
+            String sessionToken,
```

Comment:
This indentation should be the same (4 spaces) as everywhere else. Check your java formatting settings in IntelliJ.